Assets/Plugins/Windows/Core/WindowsPlatformSpecifics.cs

```diff
@@ -58,9 +70,7 @@ public class WindowsPlatformSpecifics : PlatformSpecifics<WindowsConfig>
 #if ENABLE_CONFIGURE_STEAM_FROM_MANAGED
         private static readonly string SteamDllName = 
 #if UNITY_64
-        "steam_api64.dll";
-#else
-        "steam_api.dll";
```

Comment:
What happened to the steam DLL name?

Reply:
Ah - yes good call. Combination of merge conflict and accidental removal of the definition here. It has now been restored and resolved.